corehq/apps/importer/const.py

```diff
@@ -1,2 +1,11 @@
 class LookupErrors:
     NotFound, MultipleResults = range(2)
+
+
+class ImportErrors:
+    InvalidOwnerName = 'Invalid Owner Name'
+    InvalidOwnerId = 'Invalid Owner ID'
+    InvalidParentId = 'Invalid Parent ID'
+    InvalidDate = 'Invalid Date'
+    BlankExternalId = 'Blank External ID'
+    CaseGeneration = 'Case Generation Error'
```

Comment:
@millerdev is right - you should use either `ugettext_lazy` or `ugettext_noop` here to mark them for static analysis.